src/pages/index.js

```diff
@@ -24,24 +20,48 @@ const App = () => {
     setPreview(!preview);
   };
 
+  const deleteCVHandler = () => {
+    storage.deleteResume(localStorage);
+    updateData({});
+  };
+
+  const store = {
+    preview: { get: preview, set: setPreview },
+    data: { get: data, set: updateData },
+    deleteCV: deleteCVHandler,
+  };
+
+  useEffect(() => {
+    if (localStorage.getItem('resume')) {
```

Comment:
place resume in a constant.
maybe in file `constants/storage.js`

Reply:
done